meeshkan/server/server/callbacks.py

```diff
@@ -80,16 +80,7 @@ def _map_results(self, response, result, format, response_type):
     def __call__(self, request, response):
         callback = self._callbacks.get((request.host, request.method.value, request.pathname))
         if callback is not None:
-            sink = StringIO()
-            #### there is a bug where sometimes, for unclear reasons,
-            #### the request body is sometimes written as a byte string
-            #### instead of a string. we hackishly fix here, but note that
-            #### the bug exists. would be good to know why
-            request = replace(request, body=str(request.body))
-            HttpExchangeWriter(sink).write(HttpExchange(request=request, response=response))
-            sink.seek(0)
-            res = json.loads('\n'.join([x for x in sink]))
-            out = callback(res['request'], res['response'], storage_manager.default)
```

Comment:
Could this bug have been fixed by https://github.com/meeshkan/meeshkan/commit/36d5f4597637170d7785e42132adf2b129645e09#diff-cc24a3db2500505592b6c0ea4f93ec18L65 ?